libkbfs/tlf_journal.go

```diff
@@ -365,11 +365,11 @@ func makeTLFJournal(
 
 	// Do this only once we're sure we won't error.
 	storedBytes := j.blockJournal.getStoredBytes()
+	j.diskLimiter.onJournalEnable(storedBytes)
 	if storedBytes > 0 {
-		availableBytes := j.diskLimiter.ForceAcquire(storedBytes)
 		j.log.CDebugf(ctx,
-			"Force-acquired %d bytes for %s: available=%d",
-			storedBytes, tlfID, availableBytes)
+			"Force-acquired %d bytes for %s: %v",
```

Comment:
My comment from the other PR applies here -- I feel like we should be printing out `storedBytes`, not `j.diskLimiter` here.  But I might be missing something.